lib/origen/remote_manager.rb

```diff
@@ -119,8 +119,46 @@ def symlink?(path)
       end
     end
 
+    def resolve_remotes!
```

Comment:
Can you add some # comments here as to what the method does and when/why you would call it?

After that, this is good to go from my perspective.

Thanks!